go/def.bzl

```diff
@@ -23,6 +23,8 @@ load(
 load(
     "@io_bazel_rules_go//go/private:providers.bzl",
     _GoLibrary = "GoLibrary",
+    _GoSource = "GoSource",
+    _GoPath = "GoPath",
```

Comment:
Feel free to expose `GoArchive` and `GoArchiveData`, too. They're documented and fairly stable at this point.

Reply:
awesome